react/components/TrapFocus.tsx

```diff
@@ -5,7 +5,7 @@ interface Props {
   open: boolean
 }
 
-const TrapFocus: React.FC<Props> = props => {
+const TrapFocus: React.FC<Props> = (props) => {
```

Comment:
Same here

Reply:
Shouldn't convert to function? Same reason that i left before